packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-blocks-manage-button.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+import { MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { isReusableBlock } from '@wordpress/blocks';
+import { useSelect } from '@wordpress/data';
+import { BlockSettingsMenuControls } from '@wordpress/block-editor';
+import { addQueryArgs } from '@wordpress/url';
+
+function ReusableBlocksManageButton( { clientId } ) {
+	const { isVisible } = useSelect(
+		( select ) => {
+			const { getBlock } = select( 'core/block-editor' );
+			const { canUser } = select( 'core' );
+			const blockObj = getBlock( clientId );
+
+			const reusableBlock =
+				blockObj && isReusableBlock( blockObj )
+					? select( 'core' ).getEntityRecord(
+							'postType',
+							'wp_block',
+							blockObj.attributes.ref
+					  )
+					: null;
+
+			return {
+				isVisible:
+					!! reusableBlock &&
+					!! canUser( 'update', 'blocks', reusableBlock.id ),
+			};
+		},
+		[ clientId ]
+	);
+
+	if ( ! isVisible ) {
+		return null;
+	}
+
+	return (
+		<BlockSettingsMenuControls>
+			<MenuItem
+				onClick={ () => {
+					window.location = addQueryArgs( 'edit.php', {
```

Comment:
I think this menu item should ideally render as a link (an `<a>` element instead of a `<button>`). 

`MenuItem` can take the same props as `Button`, so it should be possible to pass the value to the `href` prop:
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/button/index.js#L23

I do wonder if it should open in a new tab 🤔

Reply:
Good to know this! I used your suggestion in my most recent commit. 

Regarding the new tab functionality, I'm totally open to changing it but for now I'm leaving the functionality consistent with the "Manage all reusable blocks" button shown below. It doesn't open a new tab.

![image](https://user-images.githubusercontent.com/9662992/99609494-dfccc580-29c4-11eb-8455-a5276f23bb37.png)